### ISSUE TITLE
Error when running the github action: [tool.poetry] section not found in /github/workspace/pyproject.toml

### ISSUE THREAD

**Issue by Kajiih** [kvankova/code-embedder#48]
### Checks

- [X] I have checked that this issue has not already been reported.
- [X] I have confirmed this bug exists on the [latest version](https://github.com/kvankova/code-embedder) of Code-Embedder.

### Reproducible example

The error happens when using the github action with the following workflow file (in a project without `poetry`):
```yaml
name: Code Embedder Test

on: push

permissions:
  contents: write

jobs:
  code_embedder:
    name: "Code embedder"
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v4

      - name: Run code embedder
        uses: kvankova/code-embedder@v1.1.0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

### Log output

```shell
Run kvankova/code-embedder@v1.1.0
/usr/bin/docker run --name b8b24065e8f5526cdf49a3b654dd6e5ed89232_f98635 --label b8b240 --workdir /github/workspace --rm -e "PYTHON_VERSION" -e "UV_SYSTEM_PYTHON" -e "GITHUB_TOKEN" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_RESULTS_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/nested_dict_tools/nested_dict_tools":"/github/workspace" b8b240:65e8f5526cdf49a3b654dd6e5ed89232
From https://github.com/Kajiih/nested_dict_tools
 * branch            main       -> FETCH_HEAD
Already up to date.
Searching for code snippets...

[tool.poetry] section not found in /github/workspace/pyproject.toml
```

### Issue description

I'm getting this error:
```[tool.poetry] section not found in /github/workspace/pyproject.toml```

Is using poetry/having a `[tool.poetry]` section necessary?

### Expected behavior

Expected no exception

**Comment by Kajiih** [comment]
I temporarily fixed it by adding a dumb section in the `pyproject.toml`
```toml
[tool.poetry]
name = ""
version = "0.0.1" # Looks like to be necessary
description = ""
authors = [" <>"]
```

Related to poetry, I had errors when I was using this GitHub action with a virtual environment, like the one created by `uv`, for example by installing the dependencies with this step:
```yaml
- name: Install the project dependencies
        run: uv sync --all-extras --dev
```

I need to install the dependencies because I am performing some formatting before embedding the code.

It works if I don't use `uv` but pip alone, but it is not as practical. So with a step like:
```yaml
- name: Install the project dependencies
        run: pip install .
```

**Comment by kvankova** [comment]
Hey:) thanks for raising this issue. I have fixed it in PR #49 and released a new version of the package [1.1.1](https://github.com/kvankova/code-embedder/releases/tag/v1.1.1). I have tested it on a package without `poetry` and it should be now working. Try it out and feel free to close this. :)

**Comment by Kajiih** [comment]
Thank you!
I tested on my side, and it works as well, including when using uv!
